YAXLib/YAXLib.csproj

```diff
@@ -19,6 +19,7 @@
     </ItemGroup>
     
     <ItemGroup>
+        <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="3.1.6" />
```

Comment:
For what is this needed? I've removed this in #122.

Reply:
Only required in case support for net46 should be supported.